docs/architecture/go_components.md

```diff
@@ -92,27 +92,21 @@ A Go component's coordinates will also need to consist of a type, provider, name
 
 * **type** should be go
-* **provider** should be "proxy.golang.org" (we may need to add other providers in the future)
-* **namespace** should be anything that appears before the package name
+* **provider** should be the import path (if any)
+* **namespace** should be anything that appears before the package name (it should be "-" if nothing appears before the package name)
 * **name** should be the name of the package
 * **revision** should be the component revision (note - go components seem to have multiple ways of specifying revisons - sometimes version numbers, sometimes commit hashes)
 
-Like with Maven components, we translate namespaces that have multiple slashes (like `golang.org/x`) into namepaces with periods (`golang.org.x`).
+Like with Maven components, we translate namespaces that have multiple slashes (like `Azure/azure-event-hubs-go`) to ones with periods (`Azure.azure-event-hubs-go`).
 
-For example: `go.uber.org/multierr v1.6.0` maps to `go/proxy.golang.org/go.uber.org/multierr/v1.6.0`
+### Examples 
 
-**Examples that seem to work well:**
-
-* `collectd.org v0.5.0` maps to `go/proxy.golang.org/-/collectd.org/v0.5.0`
-* `go.starlark.net v0.0.0-20210406145628-7a1108eaa012` maps to `go/proxy.golang.org/go.starlark.net/v0.0.0-20210406145628-7a1108eaa012`
-* `cloud.google.com/go v0.56.0` maps to `go/proxy.golang.org/cloud.google.com/go/v0.56.0`
-* `code.cloudfoundry.org/clock v1.0.0` maps to `go/proxy.golang.org/code.cloudfoundry.org/clock/v1.0.0`
-* `go.uber.org/multierr v1.6.0` maps to `go/proxy.golang.org/go.uber.org/multierr/v1.6.0`
-* `k8s.io/api v0.20.4` maps to `go/proxy.golang.org/k8s.io/api/v0.20.4`
-* `modernc.org/sqlite v1.10.8` maps to `go/proxy.golang.org/modernc.org/sqlite v1.10.8`
-* `github.com/project/package v1.2.3` maps to `go/proxy.golang.org/github.com.project/package/v1.2.3`
-
-**Examples that seem a little weird:**
-* `github.com/Azure/azure-event-hubs-go/v3 v3.2.0` maps to `go/proxy.golang.org/github.com.Azure.azure-event-hubs-go/v3/v3.2.0`
-* `golang.org/x/net v0.0.0-20210226172049-e18ecbb05110` maps to `go/proxy.golang.org/golang.org.x/net/v0.0.0-20210226172049-e18ecbb05110`
+* `collectd.org v0.5.0` maps to `go/-/-/collectd.org/v0.5.0`
+* `go.starlark.net v0.0.0-20210406145628-7a1108eaa012` maps to `go/-/-/go.starlark.net/v0.0.0-20210406145628-7a1108eaa012`
+* `cloud.google.com/go v0.56.0` maps to `go/cloud.google.com/-/go/v0.56.0`
+* `code.cloudfoundry.org/clock v1.0.0` maps to `go/code.cloudfoundry.org/-/clock/v1.0.0`
+* `go.uber.org/multierr v1.6.0` maps to `go/go.uber.org/-/multierr/v1.6.0`
+* `github.com/project/package v1.2.3` maps to `go/github.com/project/package/v1.2.3`
+* `golang.org/x/net v0.0.0-20210226172049-e18ecbb05110` maps to `go/golang.org.x/net/v0.0.0-20210226172049-e18ecbb05110`
```

Comment:
Should be `go/golang.org/x/net/v0.0.0-20210226172049-e18ecbb05110` i think

Reply:
ah- typo!